### ISSUE TITLE
Shell access

### ISSUE THREAD

**Issue by jkroepke** [kubernetes/dashboard#8226]
### What would you like to be added?

I would like it see a web shell where I can execute kubectl commands.

### Why is this needed?

Sometimes, its easier to debug issues with kubectl.

**Comment by maciaszczykm** [comment]
I don't think that this will be high on our priority list to make it work out of the box. If you want you can use one of existing kubectl containers (i.e. https://hub.docker.com/r/bitnami/kubectl) and then use exec feature which dashboard already provides.
